sw/airborne/modules/computer_vision/video_thread.c

```diff
@@ -98,8 +100,13 @@ static void *video_thread_function(void *data)
 
   // create the images
   if (vid->filters & VIDEO_FILTER_DEBAYER) {
-    // fixme: don't hardcode size, works for Bebop front camera for now
-#define IMG_FLT_SIZE 272
+    // fixme: don't hardcode size, works for bebop front camera for now
+    #ifdef BOARD_BEBOP
```

Comment:
why not `#ifndef IMG_FLT_SIZE` followed by the current default and set a different value for minidrone in board header ?